portafly/src/pages/DeveloperAccounts.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useDocumentTitle, Loading, DeveloperAccountsTable } from 'components'
+import {
+  Alert,
+  PageSection,
+  PageSectionVariants,
+  TextContent,
+  Text
+} from '@patternfly/react-core'
+import { useGetDeveloperAccounts } from 'dal/accounts'
+import { useTranslation } from 'i18n/useTranslation'
+
+const DeveloperAccounts: React.FunctionComponent = () => {
+  const { t } = useTranslation('accounts')
+  useDocumentTitle(t('page_title'))
+
+  const { accounts, error, isPending } = useGetDeveloperAccounts()
+
+  return (
+    <>
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">{t('body_title')}</Text>
+        </TextContent>
+      </PageSection>
+
+      <PageSection>
+        {isPending && <Loading />}
+        {error && <Alert variant="danger" title={error.message} />}
+        {accounts && <DeveloperAccountsTable accounts={accounts} />}
```

Comment:
Instead of having https://github.com/3scale/porta/pull/1785/files#diff-8d46996cf806d696795c4eadfc6f2a05R14-R16 I'd rather have here a ternary operator loading `DeveloperAccountsTable` or `SimpleEmptyState`

Reply:
From my point of view, an empty state always belong to the list/table, since it features other elements such as a toolbar and pagination, and the EmptyState would replace everything. But I'd like to confirm this with @cntlsn first because there is no empty state in the page's mockups and my assumption might be wrong.

Reply:
Ok, makes sense. Would be good then to avoid repetition of the conditional check in all the pages to make it part of the Table and list components if possible